test/src/lib.rs

```diff
@@ -505,4 +505,72 @@ mod tests {
         do_flush(&conn, &tmp_db)?;
         Ok(())
     }
+
+    #[test]
+    fn test_simple_delete() -> anyhow::Result<()> {
+        let _ = env_logger::try_init();
+        let tmp_db = TempDatabase::new("CREATE TABLE test (x INTEGER PRIMARY KEY);");
+        let conn = tmp_db.connect_limbo();
```

Comment:
in main we also refactor this integration tests module a bit. can we rebase to avoid conflict here.